File: my-folder/0217-contains-duplicate/solution.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        unordered_set<int>x;
        for(int i=0;i<nums.size();i++)
            x.insert(nums[i]);
        if(x.size()==nums.size())
            return false;
        else
            return true;
        
    }
};

File: my-folder/0217-contains-duplicate/solution.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        HashSet<Integer>h=new HashSet<>();
        for(int x:nums)
        {
            if(h.contains(x))
                return true;
            h.add(x);
        }
        return false;
        
    }
}
